src/utils/index.js

```diff
@@ -9,7 +25,20 @@ export const setupDocsearch = () => {
     apiKey: "f53a424573adab20d04faa2db150c349",
     indexName: "livechatinc",
     inputSelector: "#search",
-    debug: false
+    debug: false,
+    handleSelected: function (input, event, suggestion, datasetNumber, context) {
+      if (
+        context.selectionMethod === "click" ||
+        context.selectionMethod === "enterKey"
+      ) {
+        logAmplitudeEvent("User selected suggestion and input", {
+          url: suggestion.url,
+          input: input.getVal()
+        });
+
+        window.location.href = suggestion.url;
```

Comment:
This would cause the page to reload, and possibly the event not being sent (not sure if Amplitude SDK can handle such cases and safely finish sending the event after the reload).

I'd be curious if we could use [Gatsby navigate](https://www.gatsbyjs.org/docs/gatsby-link/#how-to-use-the-navigate-helper-functionc) instead. It would also provide better experience for the user.

Reply:
As far as I know the page always reload on search there.

Reply:
The events are being sent : )

Reply:
Changed to Gatsby navigate